server/form-pages/apply/required-referrals/accommodation-referral-details/dtrDetails.ts

```diff
@@ -93,17 +121,39 @@ export default class DtrDetails implements TasklistPage {
     }
 
     if (dateIsBlank(this.body, 'date')) {
-      errors.date = 'You must specify the date DTR / NOP was submitted'
+      errors.date = 'Enter a submission date'
     } else if (!dateAndTimeInputsAreValidDates(this.body, 'date')) {
-      errors.date = 'You must specify a valid date DTR / NOP was submitted'
+      errors.date = 'You must specify a valid submission date'
     } else if (dateIsInFuture(this.body.date)) {
       errors.date = 'The date DTR / NOP was submitted must not be in the future'
```

Comment:
Language should also be updated here:

```suggestion
      errors.date = 'The submission date must not be in the future'
```